README.md

```diff
@@ -151,13 +151,13 @@ everything else from the plugin:
 <Chain "FilterOutDetailedDockerStats">
   <Rule "CpuUsage">
     <Match "regex">
-      Type "^cpu.usage$"
+      Type "^cpu.percent$"
```

Comment:
instead of replacing cpu.usage counts with the cpu.percent gauge, maybe add CpuPercent rule.